results/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```

Comment:
This could be formatted better. I'm sure you ran out of time, but I would suggest using tables.